lib/active_merchant/billing/gateways/beanstream/ipp_core.rb

```diff
@@ -2,28 +2,31 @@
 
 module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
-    class IppGateway < Gateway
```

Comment:
please revert the changes to this file. The `IppGateway` might be used and cannot be changed just for beanstream. Instead bring the variations to `BeanstreamIppGateway`.